cluster-provision/centos8/scripts/vm.sh

```diff
@@ -68,10 +68,8 @@ fi
 
 calc_next_disk
 
-disk_size=$(qemu-img info --output json ${last} | jq '.["virtual-size"]')
-
-echo "Creating disk \"${next} backed by ${last} with size ${disk_size}\"."
-qemu-img create -f qcow2 -o backing_file=${last} ${next} ${disk_size}
+echo "Creating disk \"${next} backed by ${last} with size 35G\"."
+qemu-img create -f qcow2 -o backing_file=${last} ${next} 35G
```

Comment:
Can use 35G as default in case the centos8 virtualbox change size (like it did for centos7) 
```bash
default_disk_size=37580963840 # 35G
disk_size=$(qemu-img info --output json ${last} | jq '.["virtual-size"]')
if [ $disk_size -lt $default_disk_size ]; then
    disk_size=$default_disk_size
fi
```

Reply:
Good idea, I'll change it to that. Thanks!

Reply:
@qinqon, done, also re-added growpart to actually use the space.